### PR TITLE
Update create-dns-records-at-godaddy.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed Godaddy DNS changes on the Doc

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/dns/create-dns-records-at-godaddy.md [hunk 1]
```diff
@@ -54,48 +54,33 @@ Before you use your domain with Office 365, we have to make sure that you own it
 > [!NOTE]
 > This record is used only to verify that you own your domain; it doesn't affect anything else. You can delete it later, if you like. 
   
-Follow the steps below or [watch the video (start at 0:43)](https://support.office.com/en-us/article/Video-Create-DNS-records-at-GoDaddy-for-Office-365-7d56924e-77ab-4ac6-b6f2-ca0e1a72249d?ui=en-US&amp;rs=en-US&amp;ad=US).
+Follow the steps below .
```
Comment (JohanFreelancer9) [inline Office365-Admin/dns/create-dns-records-at-godaddy.md]:
```suggestion
Follow the steps below.
```

File: Office365-Admin/dns/create-dns-records-at-godaddy.md [hunk 2]
```diff
@@ -54,48 +54,33 @@ Before you use your domain with Office 365, we have to make sure that you own it
...
-    ![GoDaddy-BP-Configure-1-3](../media/4823c977-3662-4057-b661-c6e393c25855.png)
-  
-4. Choose ** Add Record **.
-    
-    ![GoDaddy-BP-Configure-1-4](../media/ab66fdc2-e16a-43ab-924e-3365ac037fe0.png)
-  
-5. Choose **TXT (Text)** from the drop-down list. 
+    ![GoDaddy-BP-Configure-1-2](https://user-images.githubusercontent.com/45987684/56528038-94b6ac00-651c-11e9-8874-12db60cc7ea6.PNG)
+   
+3. Choose **Add**,
```
Comment (JohanFreelancer9) [inline Office365-Admin/dns/create-dns-records-at-godaddy.md]:
```suggestion
3. Choose **Add**.
```

File: Office365-Admin/dns/create-dns-records-at-godaddy.md [hunk 3]
```diff
@@ -54,48 +54,33 @@ Before you use your domain with Office 365, we have to make sure that you own it
...
     
-    ![GoDaddy-BP-Verify-1-0](../media/33e9173a-e046-41a2-98b7-bae8e0d5a081.png)
+    ![GoDaddy-BP-Configure-1-4](https://user-images.githubusercontent.com/45987684/56527673-ffb3b300-651b-11e9-91c2-83dc9fe5ca30.PNG)
   
-6. In the boxes for the new record, type or copy and paste the values from the following table.
-    
-    (Select the **TTL** value from the drop-down list.) 
+4. Choose **TXT (Text)** from the drop-down list, In the boxes for the new record, type or copy and paste the values from the following table.
```
Comment (JohanFreelancer9) [inline Office365-Admin/dns/create-dns-records-at-godaddy.md]:
```suggestion
4. Choose **TXT (Text)** from the drop-down list. In the boxes for the new record, type or copy and paste the values from the following table.
```

File: Office365-Admin/dns/create-dns-records-at-godaddy.md [hunk 4]
```diff
@@ -121,80 +106,59 @@ When Office 365 finds the correct TXT record, your domain is verified.
 ## Add an MX record so email for your domain will come to Office 365
 <a name="BKMK_add_MX"> </a>
 
-Follow the steps below or [watch the video (start at 3:36)](https://support.office.com/en-us/article/Video-Create-DNS-records-at-GoDaddy-for-Office-365-7d56924e-77ab-4ac6-b6f2-ca0e1a72249d?ui=en-US&amp;rs=en-US&amp;ad=US).
+Follow the steps below
```
Comment (JohanFreelancer9) [inline Office365-Admin/dns/create-dns-records-at-godaddy.md]:
```suggestion
Follow the steps below.
```

File: Office365-Admin/dns/create-dns-records-at-godaddy.md [hunk 5]
```diff
@@ -121,80 +106,59 @@ When Office 365 finds the correct TXT record, your domain is verified.
...
-  
-3. Choose **DNS Zone File**.
+2. Under **Domains**, choose DNS under the domain that you want to edit.
     
-    ![GoDaddy-BP-Configure-1-3](../media/4823c977-3662-4057-b661-c6e393c25855.png)
-  
-4. Choose **Add Record**.
+    ![GoDaddy-BP-Configure-1-2](https://user-images.githubusercontent.com/45987684/56528038-94b6ac00-651c-11e9-8874-12db60cc7ea6.PNG)
+   
+3. Choose **Add**,
```
Comment (JohanFreelancer9) [inline Office365-Admin/dns/create-dns-records-at-godaddy.md]:
```suggestion
3. Choose **Add**.
```

File: Office365-Admin/dns/create-dns-records-at-godaddy.md [hunk 6]
```diff
@@ -121,80 +106,59 @@ When Office 365 finds the correct TXT record, your domain is verified.
...
-    ![GoDaddy-BP-Configure-2-5](../media/e8aacc46-7dcb-4c94-9250-41ce21f6c71f.png)
+6. Choose **Save**.
+
+
   
 ## Add the CNAME records that are required for Office 365
 <a name="BKMK_add_CNAME"> </a>
 
-Follow the steps below or [watch the video (start at 4:27)](https://support.office.com/en-us/article/Video-Create-DNS-records-at-GoDaddy-for-Office-365-7d56924e-77ab-4ac6-b6f2-ca0e1a72249d?ui=en-US&amp;rs=en-US&amp;ad=US).
+Follow the steps below
```
Comment (JohanFreelancer9) [inline Office365-Admin/dns/create-dns-records-at-godaddy.md]:
```suggestion
Follow the steps below .
```

File: Office365-Admin/dns/create-dns-records-at-godaddy.md [hunk 7]
```diff
@@ -121,80 +106,59 @@ When Office 365 finds the correct TXT record, your domain is verified.
...
-  
-3. Choose **DNS Zone File**.
+2. Under **Domains**, choose DNS under the domain that you want to edit.
     
-    ![GoDaddy-BP-Configure-1-3](../media/4823c977-3662-4057-b661-c6e393c25855.png)
-  
-4. Choose **Add Record**.
+    ![GoDaddy-BP-Configure-1-2](https://user-images.githubusercontent.com/45987684/56528038-94b6ac00-651c-11e9-8874-12db60cc7ea6.PNG)
+   
+3. Choose **Add**,
```
Comment (JohanFreelancer9) [inline Office365-Admin/dns/create-dns-records-at-godaddy.md]:
```suggestion
3. Choose **Add**.
```

File: Office365-Admin/dns/create-dns-records-at-godaddy.md [hunk 8]
```diff
@@ -207,59 +171,39 @@ Follow the steps below or [watch the video (start at 4:27)](https://support.offi
...
-    
-    Repeat these steps until you have created all six of the CNAME records.
-    
-9. When you have added the final CNAME record, choose **Finish**.
-    
-    ![GoDaddy-BP-Configure-3-3](../media/43248c08-52bb-4dc2-9116-e4b9379496d3.png)
+
   
-10. Choose **Save Changes**.
+7. Repeat these step to add the CNAME record until you have created all six of the CNAME records.
```
Comment (JohanFreelancer9) [inline Office365-Admin/dns/create-dns-records-at-godaddy.md]:
```suggestion
6. Repeat these steps to add the next CNAME record until you have created all six of the CNAME records.
```

File: Office365-Admin/dns/create-dns-records-at-godaddy.md [hunk 9]
```diff
@@ -207,59 +171,39 @@ Follow the steps below or [watch the video (start at 4:27)](https://support.offi
...
+
   
 ## Add a TXT record for SPF to help prevent email spam
 <a name="BKMK_add_TXT"> </a>
 
 > [!IMPORTANT]
 > You cannot have more than one TXT record for SPF for a domain. If your domain has more than one SPF record, you'll get email errors, as well as delivery and spam classification issues. If you already have an SPF record for your domain, don't create a new one for Office 365. Instead, add the required Office 365 values to the current record so that you have a  *single*  SPF record that includes both sets of values. 
   
-Follow the steps below or [watch the video (start at 5:15)](https://support.office.com/en-us/article/Video-Create-DNS-records-at-GoDaddy-for-Office-365-7d56924e-77ab-4ac6-b6f2-ca0e1a72249d?ui=en-US&amp;rs=en-US&amp;ad=US).
+Follow the steps below
```
Comment (JohanFreelancer9) [inline Office365-Admin/dns/create-dns-records-at-godaddy.md]:
```suggestion
Follow the steps below.
```

File: Office365-Admin/dns/create-dns-records-at-godaddy.md [hunk 10]
```diff
@@ -207,59 +171,39 @@ Follow the steps below or [watch the video (start at 4:27)](https://support.offi
...
-  
-3. Choose **DNS Zone File**.
+2. Under **Domains**, choose DNS under the domain that you want to edit.
     
-    ![GoDaddy-BP-Configure-1-3](../media/4823c977-3662-4057-b661-c6e393c25855.png)
-  
-4. Choose **Add Record**.
+    ![GoDaddy-BP-Configure-1-2](https://user-images.githubusercontent.com/45987684/56528038-94b6ac00-651c-11e9-8874-12db60cc7ea6.PNG)
+   
+3. Choose **Add**,
```
Comment (JohanFreelancer9) [inline Office365-Admin/dns/create-dns-records-at-godaddy.md]:
```suggestion
3. Choose **Add**.
```

File: Office365-Admin/dns/create-dns-records-at-godaddy.md [hunk 11]
```diff
@@ -269,40 +213,31 @@ Follow the steps below or [watch the video (start at 5:15)](https://support.offi
...
-8. Choose **Save Changes**.
+6. Choose **Save**.
     
-    ![GoDaddy-BP-Configure-4-3](../media/475f05e1-f15c-4544-b5ce-339d15d55cee.png)
   
 ## Add the two SRV records that are required for Office 365
 <a name="BKMK_add_SRV"> </a>
 
-Follow the steps below or [watch the video (start at 5:59)](https://support.office.com/en-us/article/Video-Create-DNS-records-at-GoDaddy-for-Office-365-7d56924e-77ab-4ac6-b6f2-ca0e1a72249d?ui=en-US&amp;rs=en-US&amp;ad=US).
+Follow the steps below
```
Comment (JohanFreelancer9) [inline Office365-Admin/dns/create-dns-records-at-godaddy.md]:
```suggestion
Follow the steps below.
```

File: Office365-Admin/dns/create-dns-records-at-godaddy.md [hunk 12]
```diff
@@ -269,40 +213,31 @@ Follow the steps below or [watch the video (start at 5:15)](https://support.offi
...
-  
-3. Choose **DNS Zone File**.
+2. Under **Domains**, choose DNS under the domain that you want to edit.
     
-    ![GoDaddy-BP-Configure-1-3](../media/4823c977-3662-4057-b661-c6e393c25855.png)
-  
-4. Choose **Add Record**.
+    ![GoDaddy-BP-Configure-1-2](https://user-images.githubusercontent.com/45987684/56528038-94b6ac00-651c-11e9-8874-12db60cc7ea6.PNG)
+   
+3. Choose **Add**,
```
Comment (JohanFreelancer9) [inline Office365-Admin/dns/create-dns-records-at-godaddy.md]:
```suggestion
3. Choose **Add**.
```

**Comment by dianef77** [comment]
@pebaum can you please review this pull request and decide what action to take? Thanks.

**Comment by mypil** [comment]
@pebaum - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #413?

Thank you.

**Comment by mypil** [comment]
@pebaum - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #413?

Thank you.

**Pull request merged**